playbooks/tasks/portal-role-task.yml

```diff
@@ -138,7 +134,14 @@
 
 - name: Set Sia wallet seed defined
   set_fact:
-    sia_wallet_seed_defined: "{{ webportal_server_config.sia_wallet_seed is defined and webportal_server_config.sia_wallet_seed != None }}"
+    sia_wallet_seed_defined: >-
+      {{
```

Comment:
```suggestion
      "{{
```

Reply:
Good catch.
Actually the quotation marks should be removed from the end of the block, not from the beginning.